src/shared/components/case-viewer/case-review-specific-access-request/case-review-specific-access-request.component.spec.ts

```diff
@@ -0,0 +1,183 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement, Type } from '@angular/core';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { ActivatedRoute, ActivatedRouteSnapshot, Data, ParamMap, Params, Route, UrlSegment } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AccessManagementRequestReviewMockModel } from '../../../../app.config';
```

Comment:
Can the AccessManagementRequestReviewMockModel  be put into it's own file instead of adding it to app.config?

Reply:
possible , but this MockModel is done the same patern with other mockModels.